docs/cli/secretcli.md

```diff
@@ -174,8 +174,8 @@ flag can have a value of `sync` (default), `async`, or `block`, where `sync` mak
 the client return a CheckTx response, `async` makes the client return immediately,
 and `block` makes the client wait for the tx to be committed (or timing out).
 
-It is important to note that the `block` mode should **not** be used in most
-circumstances. This is because broadcasting can timeout but the tx may still be
+It is important to note, that the `block` mode should **not** be used in most
+circumstances because broadcasting can timeout, but the tx may still be
 included in a block. This can result in many undesirable situations. Therefore, it
```

Comment:
Should put some specific examples of 'undesirable situations. Can you add a few here? @dylanschultzie

Reply:
That'll be a phase 3 thing.